<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>364</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>473</width>
    <height>331</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HexEngine v0.1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#MainWindow { 
	border: 1px solid;
	border-radius: 9px
	background-color: none; 
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget
{
	background-color: none;
	background-image: url(:/files/background.jpg);
	font: 10pt &quot;Arial&quot;;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="5">
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame
{
	border: 2px solid black;
	border-color: rgb(85, 170, 255);
	border-radius: 14px;
	background-color: transparent;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel
{
	border: none;
	border-color: none;
	color: rgb(255, 255, 255);
	font: 10pt &quot;Arial&quot;;
}</string>
         </property>
         <property name="text">
          <string>Entrada</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btn_browse_input">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border: 2px solid;
	border-color: rgb(85, 170, 235);
	border-radius: 8px;
	background-color: rgb(0, 170, 255);
	color: rgb(255, 255, 255);
	min-height: 18px;
	min-width: 50px;
	font: 10pt &quot;Arial&quot;;
}

QPushButton::hover
{
	color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>Procurar</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel
{
	border: none;
	border-color: none;
	color: rgb(255, 255, 255);
	font: 10pt &quot;Arial&quot;;
}</string>
         </property>
         <property name="text">
          <string>Saída</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="line_edit_input">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit
{
	border: 2px solid;
	border-color: rgb(85, 170, 255);
	border-radius: 8px
}</string>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="4">
        <widget class="QTextBrowser" name="text_browser">
         <property name="styleSheet">
          <string notr="true">QTextBrowser{
	border: 2px solid black;
	border-color: rgb(85, 170, 255);
	border-radius: 6px;
	background-color: none;
	color: rgb(0, 0, 0);
	font: 75 8pt &quot;Arial&quot;;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="btn_browse_output">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border: 2px solid;
	border-color: rgb(85, 170, 235);
	border-radius: 8px;
	background-color: rgb(0, 170, 255);
	color: rgb(255, 255, 255);
	min-height: 18px;
	min-width: 50px;
	font: 10pt &quot;Arial&quot;;
}

QPushButton::hover
{
	color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>Procurar</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="line_edit_output">
         <property name="styleSheet">
          <string notr="true">QLineEdit
{
	border: 2px solid;
	border-color: rgb(85, 170, 255);
	border-radius: 8px
}</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>line_edit_input</zorder>
      <zorder>btn_browse_input</zorder>
      <zorder>label</zorder>
      <zorder>label_3</zorder>
      <zorder>line_edit_output</zorder>
      <zorder>text_browser</zorder>
      <zorder>btn_browse_output</zorder>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QProgressBar" name="progress_bar">
      <property name="styleSheet">
       <string notr="true">QProgressBar
{
	border: 2px solid;
	border-color: rgb(0, 170, 0);
	color: rgb(0, 0, 0);
	text-align: center;
	background-color: none;
	border-radius: 8px;
	font: 10pt &quot;Arial&quot;;
}</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btn_init_engine">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	padding: 1px;
	border: 2px solid;
	border-color: rgb(0, 170, 0);
	border-radius: 8px;
	background-color: rgb(0, 150, 0);
	color: rgb(255, 255, 255);
	min-height: 18px;
	min-width: 60px;
	font: 10pt &quot;Arial&quot;;
}

QPushButton::hover
{
	color: rgb(0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>Iniciar</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btn_stop_engine">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	padding: 1px;
	border: 2px solid;
	border-color: rgb(0, 170, 0);
	border-radius: 8px;
	background-color: rgb(0, 150, 0);
	color: rgb(255, 255, 255);
	min-height: 18px;
	min-width: 60px;
	font: 10pt &quot;Arial&quot;;
}

QPushButton::hover
{
	color: rgb(0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>Parar</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btn_configure_engine">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	padding: 1px;
	border: 2px solid;
	border-color: rgb(255, 0, 0);
	border-radius: 8px;
	background-color: rgb(215, 0, 0);
	color: rgb(255, 255, 255);
	min-height: 18px;
	min-width: 60px;
	font: 10pt &quot;Arial&quot;;
}

QPushButton::hover
{
	color: rgb(0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>Configurações</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>554</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuFerramentas">
    <property name="title">
     <string>Ferramentas</string>
    </property>
    <addaction name="actionBatchFileInjector"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFerramentas"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">QStatusBar
{
	background-color: rgb(0, 170, 0);
	color: rgb(255, 255, 255);
	font: 10pt &quot;Arial&quot;;
}</string>
   </property>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionBatchFileInjector">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/files/shell32_153.ico</normaloff>:/files/shell32_153.ico</iconset>
   </property>
   <property name="text">
    <string>BatchFileInjector</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
